src/smart-components/role/RolesTable.tsx

```diff
@@ -207,8 +238,13 @@ const RolesTable: React.FunctionComponent<RolesTableProps> = ({ selectedRole })
                 onPerPageSelect={onPerPageSelect}
               />
             }
+            filters={
+              <DataViewFilters onChange={(_e, values) => onSetFilters(values)} values={filters} ouiaId={`${ouiaId}-filters`}>
+                <DataViewTextFilter filterId="display_name" title="Name" placeholder="Filter by name" ouiaId={`${ouiaId}-name-filter`} />
+              </DataViewFilters>
+            }
```

Comment:
```suggestion
filters={
              <DataViewTextFilter
                filterId="display_name"
                title="Name"
                placeholder="Filter by name"
                ouiaId={`${ouiaId}-name-filter`}
                onChange={(_e, value) => onSetFilters({ display_name: value })}
              />
            }
```
We don't need the dropdown since there is just one filter for now